### PR TITLE
base variables and mixins

### PR DESCRIPTION
**Pull request by erizocosmico**
All the basic colors, texts and mixins provided in the spec of the design. (There might be more when we get to specific parts).

### CODE REVIEW

File: hugo/layouts/partials/head.html [hunk 1]
```diff
@@ -22,7 +22,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, user-scalable=yes">
 
-    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700|Roboto:300,400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet">
```
Comment (Serabe) [inline hugo/layouts/partials/head.html]: Maybe in just one line:

`https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700|Roboto:400,700`

File: hugo/layouts/partials/head.html [hunk 2]
```diff
@@ -22,7 +22,8 @@
...
     <link rel="stylesheet" href="//cdn.jsdelivr.net/font-hack/2.020/css/hack.min.css">
     <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick.css"/>
```
Comment (Serabe) [inline hugo/layouts/partials/head.html]: Do we still need slick?

File: hugo/layouts/partials/head.html [hunk 3]
```diff
@@ -22,7 +22,8 @@
...
     <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick-theme.css"/>
```
Comment (Serabe) [inline hugo/layouts/partials/head.html]: Same as above.

File: src/sass/shared/skeleton.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    box-sizing: border-box;
+}
+
+html, body {
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    font-family: $font-source-sans-pro;
```
Comment (Serabe) [inline src/sass/shared/skeleton.scss]: I rather have a variable named about what it is and not about its value in here.

You can have something like:

```scss
$font-family-source-sans-pro: 'Source Sans Pro', sans-serif;
$font-family-default: $font-family-source-sans-pro;
```

File: src/sass/shared/typefaces.scss [hunk 5]
```diff
@@ -5,3 +5,19 @@
...
+  }
+}
+
+@mixin link-dark-bg() {
+  color: white;
+
+  &:hover {
+    color: $color-pale-blue;
+  }
+}
```
Comment (Serabe) [inline src/sass/shared/typefaces.scss]: I don't think this piece of content should be in typefaces. Maybe mixins?

File: src/sass/shared/typefaces.scss [hunk 6]
```diff
@@ -5,3 +5,19 @@
   font-weight: normal;
   font-style: normal;
 }
+
+@mixin link-light-bg() {
+  color: $color-light-blue;
```
Comment (Serabe) [inline src/sass/shared/typefaces.scss]: Same as with the `$font-source-sans-pro`.

File: src/sass/shared/typefaces.scss [hunk 7]
```diff
@@ -5,3 +5,19 @@
...
+
+  &:hover {
+    color: $color-dark-blue;
```
Comment (Serabe) [inline src/sass/shared/typefaces.scss]: Same as with the `$font-source-sans-pro`.

File: src/sass/shared/typefaces.scss [hunk 8]
```diff
@@ -5,3 +5,19 @@
...
+  }
+}
+
+@mixin link-dark-bg() {
+  color: white;
```
Comment (Serabe) [inline src/sass/shared/typefaces.scss]: Extract variable.

File: src/sass/shared/typefaces.scss [hunk 9]
```diff
@@ -5,3 +5,19 @@
...
+
+  &:hover {
+    color: $color-pale-blue;
```
Comment (Serabe) [inline src/sass/shared/typefaces.scss]: Same as with the `$font-source-sans-pro`.

File: hugo/layouts/partials/head.html (hunk 1 above)
Comment (dpordomingo) [inline hugo/layouts/partials/head.html]: If we use `http` two different downloads will be slower than only one,  won't they?

File: src/sass/shared/skeleton.scss [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+    font-size: $text-size-body;
+}
```
Comment (dpordomingo) [inline src/sass/shared/skeleton.scss]: EOF
We could maybe use the `.editorconfig` used for Platform project...

File: src/sass/shared/variables.scss [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+// Text sizes
+$text-size-body: 20px;
+$text-size-menu: 24px;
+$text-size-submenu: 21px;
+$text-size-card-titles: 21px;
+$text-size-h1: 48px;
+$text-size-h2: 30px;
+$text-size-h3: 24px;
+$text-size-footer-titles: 21px;
+$text-size-footer-text: 16px;
```
Comment (Serabe) [inline src/sass/shared/variables.scss]: I rather this be `$font-size`

File: src/sass/shared/typefaces.scss (hunk 5 above)
Comment (dpordomingo) [inline src/sass/shared/typefaces.scss]: EOF

File: src/sass/shared/variables.scss [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+// Font families
+$font-roboto: 'Roboto', sans-serif;
+$font-source-sans-pro: 'Source Sans Pro', sans-serif;
+$font-source-code: 'Hack', 'monaco', monospace;
```
Comment (dpordomingo) [inline src/sass/shared/variables.scss]: EOF

File: src/sass/shared/skeleton.scss [hunk 13]
```diff
@@ -0,0 +1,15 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+html, body {
+    padding: 0;
+    margin: 0;
+}
```
Comment (dpordomingo) [inline src/sass/shared/skeleton.scss]: This and the next one seems to be a Reset behavior.

File: src/sass/shared/typefaces.scss (hunk 6 above)
Comment (dpordomingo) [inline src/sass/shared/typefaces.scss]: There is no more styles in links?
Maybe underlying, cursor...

File: src/sass/shared/typefaces.scss (hunk 8 above)
Comment (dpordomingo) [inline src/sass/shared/typefaces.scss]: Var please

File: hugo/layouts/partials/head.html (hunk 2 above)
Comment (dpordomingo) [inline hugo/layouts/partials/head.html]: If it is needed a slide effect, slick will make it easy

File: src/sass/shared/skeleton.scss (hunk 4 above)
Comment (dpordomingo) [inline src/sass/shared/skeleton.scss]: Please, do not use var values in bar names.-

File: src/sass/shared/typefaces.scss (hunk 6 above)
Comment (dpordomingo) [inline src/sass/shared/typefaces.scss]: No values in names

File: hugo/layouts/partials/head.html (hunk 2 above)
Comment (erizocosmico) [inline hugo/layouts/partials/head.html]: Until we know for sure the carousel will not be a carousel, yes.

File: hugo/layouts/partials/head.html (hunk 1 above)
Comment (Serabe) [inline hugo/layouts/partials/head.html]: Even using `https` _might_ be slower.

File: src/sass/shared/variables.scss [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+$color-links-darkbg-hover: $color-pale-grey;
+
+// Text sizes
+$font-size-body: 20px;
+$font-size-menu: 24px;
+$font-size-submenu: 21px;
+$font-size-card-titles: 21px;
+$font-size-h1: 48px;
+$font-size-h2: 30px;
+$font-size-h3: 24px;
```
Comment (dpordomingo) [inline src/sass/shared/variables.scss]: what if we want to use the h1 size over an h3?
I'd define the vars using only semantics like:
```
$font-size-title-L: 48px;
$font-size-title-M: 30px;
$font-size-title-S: 24px;
```

**Comment by erizocosmico** [comment]
Done @Serabe @dpordomingo

**Pull request merged**